react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { clicked: '' };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    const className = event.target.className;
+    if (className === 'html' || className === 'css' || className === 'js') {
+      if (this.state.clicked !== className) {
+        this.setState({ clicked: event.target.className });
+      } else {
+        this.setState({ clicked: '' });
+      }
+    }
```

Comment:
This is too specific and only works with exactly 3 topics, this should be set dynamically, below is a possible solution:

```jsx
handleClick(id) {
  this.setState({
    clicked: id
  });
}
```